src/basic/lines.js

```diff
@@ -129,14 +129,14 @@ Line.getDraw = (c3ss, id) => {
  *
  * @returns default style configuration for lines
  */
-Line.getStyle = function(c3ss, id) {
+Line.getStyle = function(c3ss, id, ord) {
   let style = {};
 
   style['lines_' + id] = {
     base: 'lines',
     blend: getBlending(c3ss),
     dash: getDashed(c3ss),
-    blend_order: 2
+    blend_order: isFinite(ord) ? ord + 1 : 1
```

Comment:
Why do we have a different initialization of `blend_order` for lines? Other symbolizers have `blend_order: ord || 1`.